src/unit_tests/schema/get_schema_from_bucket_test.py

```diff
@@ -26,6 +26,32 @@ def test_get_schema_from_bucket_200_response(test_client):
     assert response.json() == schema_test_data.test_schema_response
 
 
+def test_get_latest_schema_from_bucket_without_version(test_client):
+    """
+    When the schema is queried without version no, the schema of latest version
+    should be returned with a 200 status code
+    """
+    mock_firebase_repository = SchemaFirebaseRepository()
+    mock_stream_obj = MagicMock()
+    mock_stream_obj.to_dict.return_value = {"schema_location": "test_location"}
+    mock_firebase_repository.schemas_collection.where().order_by().limit().stream.return_value = [
+        mock_stream_obj
+    ]
+
+    tmp_storage = SchemaBucketRepository.get_schema_file_as_json
+    SchemaBucketRepository.get_schema_file_as_json = MagicMock()
+    SchemaBucketRepository.get_schema_file_as_json.return_value = (
+        schema_test_data.test_schema_response
+    )
```

Comment:
How come you're mocking all the stream logic in here? Ideally we should just mock the repository functions to avoid this, I think it makes the tests confusing.